css/style.scss

```diff
@@ -1081,54 +1081,60 @@ footer {
   .second__player {
     height: 390px;
   }
-
-  padding-top: 15px;
-  iframe {
+  .playPauseIcon {
+    position: absolute;
+    top: 0;
     width: 100%;
     height: 100%;
-  }
-  &::before {
-    color: #f44336;
-    content: "\f144";
-    font-family: "Font Awesome 5 Free";
-    font-size: 50px;
-    position: absolute;
-    left: 0;
-    right: 0;
-    text-align: center;
-    top: 50%;
-    transform: translateY(-50%);
-    transition: all 0.5s linear 0s;
-    z-index: 12;
-    opacity: 0;
-    visibility: hidden;
-  }
-  &::after {
-    color: #fff;
-    content: "\f144";
-    font-family: "Font Awesome 5 Free";
-    font-size: 50px;
-    position: absolute;
-    left: 0;
-    right: 0;
-    text-align: center;
-    top: calc(50% - 22px);
-
-    transition: all 0.3s linear 0s;
-    z-index: 11;
-  }
-  &:hover {
-    &::before {
-      visibility: visible;
-      opacity: 1;
-    }
-    &::after {
-      visibility: hidden;
-      transform: rotate(-360deg);
-      opacity: 0.0001;
+    .playControl {
```

Comment:
Better to have style to make the cursor as a pointer:
 cursor: pointer;

Reply:
Done.